modules/oc-adm-by-example-content.adoc

```diff
@@ -340,31 +340,72 @@ Create a new project
 
 
 
+== oc adm node-image create
+Create an ISO image for booting the nodes to be added to the target cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Create the ISO image and download it in the current folder
+  oc adm node-image create
+  
+  # Use a different assets folder
+  oc adm node-image create --dir=/tmp/assets
+  
+  # Specify a custom image name
+  oc adm node-image create --o=my-node.iso
```

Comment:
```./oc adm node-image create --o=my-node.iso
error: unknown flag: --o
See 'oc adm node-image create --help' for usage```

Reply:
I'll submit a PR to fix this

Reply:
Submitted https://github.com/openshift/oc/pull/1874 to fix this in the oc repo for next time. Also manually fixed it here in this PR